### DISCUSSION TITLE
Sim Speed: Convex hull vs Full rendering

### DISCUSSION THREAD

**Discussion by rohit-kumar-j** [google-deepmind/mujoco#341] [General]
I have noticed that when using convex hull rendering, in `simulate.cc`, the robot model orbits, pans, and zooms faster than with the fully rendered model.(especially robots with heavy stl files).

During training or working with time-critical computation, is it faster when we train with only convex hulls enabled when we call `mujoco.mj_step(model, data)` without visualization?

```python
while not done:
    mujoco.mj_step(model, data)
    # viewer.render()  # renders in a window
```

**Comment by yuvaltassa** [comment]
Is you are not rendering during training, it should not matter.

That said, you might get some milage from the compiler's [discardvisual](https://mujoco.readthedocs.io/en/latest/XMLreference.html#compiler) flag as that actually removes non colliding geoms and would save you some kinematics computation.
